samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -91,11 +91,12 @@ CamResolutionsTest.prototype = {
     var call = new Call();
     call.pc1.addStream(stream);
     call.establishConnection();
-    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 1000);
-    setTimeoutWithProgressBar( function() {
+    this.collectStatsStartTime = Date.now();
```

Comment:
I know, but it's complaining about stuff I haven't changed as well. New grunt criteria might have been added as well, will sort it out.